cssVariables.css

```diff
@@ -0,0 +1,85 @@
+/**
+ * Hi, sorry yes i know this is confusing.
+ *
+ * This file currently is not the actual source of the used css variables in the UI, as we have packages/build-essentials/src/styles/styleConstants.js
+ * So why do i say currently? can i look into the future? As a matter of fact yes.
+ * We are currently in the Neos Ui branch 7.3 8.0 8.1 or 8.2,
+ * beginning with Neos 8.3 we will start to use this file here as actual source of truth (see https://github.com/neos/neos-ui/pull/3366)
```

Comment:
As the future source of truth, we should clean up the values a bit to only use lowercase letters in the hex values.
And organise the various groups of variables a bit with short comments.

Then its easier to link the file from the docs and have the source explain itself mostly.
Also make it more explicit in the text that this is `@api`.

I think you should also remove the personal text and just keep the second part from Line 17 "Global...".
Git can already tell the story.

Reply:
Then why not mention this instead in the code. 
And when we introduce the new names we most likely keep the old ones as deprecated variables anyway for a while. So we don't hurt ourselves.

Reply:
Yes makes 👍